File: VB/UseStandardReportWizard/Form1.vb
Imports System.Windows.Forms
Imports DevExpress.XtraReports.Wizards

Namespace UseStandardReportWizard
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()

            ' Replace the default ReportWizardSettings service with a custom one.
            reportDesigner1.RemoveService(GetType(ReportWizardSettings))
            reportDesigner1.AddService(GetType(ReportWizardSettings), New ReportWizardSettings() With {.UseMasterDetailReportWizard = False})
        End Sub
    End Class
End Namespace

File: CS/UseStandardReportWizard/Form1.cs
using System.Windows.Forms;
using DevExpress.XtraReports.Wizards;

namespace UseStandardReportWizard {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();

            // Replace the default ReportWizardSettings service with a custom one.
            reportDesigner1.RemoveService(typeof(ReportWizardSettings));
            reportDesigner1.AddService(typeof(ReportWizardSettings), new ReportWizardSettings() {
                UseMasterDetailReportWizard = false
            });
        }
    }
}
